config/default.go

```diff
@@ -13,7 +13,7 @@ var defaultConfig = Config{
 	},
 	RunnerOptions: RunnerOptions{
 		Concurrency:   1,
-		Requests:      -1,
+		Requests:      -1, // Use GlobalTimeout as exit condition if omitted.
```

Comment:
Nitpick-but-not-a-nitpick: explicit `-1` results is the use of global timeout as exit condition. Omitting the options results in defaulting to `-1` thus the global timeout.

Reply:
Isn't it already implied given this comment is on `defaultConfig`?
On a side note we really need to rethink our defaults, I don't think infinite requests by default is a great idea, especially with a default timeout of `30s` 🤔  `100` would be more appropriate

Reply:
I agree that a limited default value would actually be safer.